snap7/bin/snap7-server.py

```diff
@@ -8,10 +8,8 @@
 import snap7
 import sys
 
-
-logging.basicConfig()
-logger = logging.getLogger()
-logger.setLevel(logging.INFO)
+logging.basicConfig(level=logging.DEBUG)
```

Comment:
i dont think we should change the default logging level for the server. I'm not sure why this would increase the code coverage number? These things are unrelated as far as I know.

Reply:
I thought that coverage wouldn't check the line if the logger level was set higher.

Reply:
no worries, but this is untrue. Maybe some code inside the logging module will not be executed but that isn't included in our code coverage report.